### PR TITLE
PR for discussions for adding artifactType

### PR DESCRIPTION
**Pull request by sajayantony**

### CODE REVIEW

File: artifact.md [hunk 1]
```diff
@@ -15,11 +15,15 @@ For the media type(s) that this is compatible with see the [matrix](media-types.
 
   This property MUST be used and contain the media type `application/vnd.oci.artifact.manifest.v1+json`.
 
-- **`blobs`** *string*
+- **`artifactType`** *string*
 
-  This OPTIONAL property contains a list of [descriptors](descriptor.md). Each descriptor represents an artifact of any IANA mediaType. The list MAY be ordered for certain artifact types like scan results.
+  This REQUIRED property specifies the mediaType of the referenced content and MAY be registered with [IANA][iana].
```
Comment (sajayantony) [inline artifact.md:20]: Whether this is  OPTIONAL or REQUIRED needs to be discussed on the call.

Comment (jdolitsky) [reply artifact.md:20]: I believe it was previously determined to be required based on preventing recent mediatype security issue

**Review by jdolitsky — APPROVED** [review]

File: descriptor.md [hunk 2]
```diff
@@ -51,8 +51,15 @@ The following fields contain the primary properties that constitute a Descriptor
   The decoded data MUST be identical to the referenced content and SHOULD be verified against the [`digest`](#digests) and `size` fields by content consumers.
   See [Embedded Content](#embedded-content) for when this is appropriate.
 
+- **`artifactType`** *string*
+  
+  This OPTIONAL property contains the [IANA][iana] mediaType of the object when the descriptor is 
+  pointing to an [artifact manifest](artifact.md).
```
Comment (sudo-bmitch) [inline descriptor.md:57]: Nit, single line per sentence.

File: artifact.md (hunk 1 above)
Comment (sudo-bmitch) [reply artifact.md:20]: I lean towards `SHOULD` instead of `REQUIRED`

**Comment by sajayantony** [comment]
Make artifactType SHOULD as per discussion.

**Comment by sajayantony** [comment]
Closing this PR and reopening another one.

**Pull request closed without merging**